app/views/agents/motifs/_form.html.slim

```diff
@@ -14,6 +14,13 @@
         .col-md-4= f.input :default_duration_in_min
         .col-md-4= f.input :color, as: "color"
 
+  - if Flipflop.vad?
```

Comment:
now that I see it here I'm wondering if `visite_a_domicile` wouldn't be a more explicit feature name and reasonably long